faq/how-do-i-change-the-colors.md

```diff
@@ -0,0 +1,29 @@
+# How do I change the colors?
+
+This article is a stub.
+
+## Update the CSS with the new colors
+
+### Relevant files
+
+- ``frontend/style/settings/_variables.scss``
+
+
+For this step, you'll need a program that can compile SCSS files into CSS. For
+example, on Ubuntu you can use ``pysassc``:
+
+```
+sudo apt install pysassc
+```
+
+From the ``frontend`` root directory, run:
+
+```
+pysassc --style=compressed --sourcemap style/rsd.scss static/style/rsd.scss.css
```

Comment:
In the `frontend/README.md` `sassc` is used instead of `pysassc`, are these equivalent?

Reply:
According to @Tommos, he used sassc from the Debian repos. However, I could only find the libsass library, but not what the libsass peeps call an implementer in my Ubuntu repos, so I used pysassc which was listed.

I do find the without-docker command easier to understand, but it does limit the user to debian/ubuntu flavor linux. 

note: ``pip install libsass`` also provides a ``sassc``

Also, I looked at install from npmjs, maybe [``node-sass``](https://www.npmjs.com/package/node-sass) could be an option (apparently, node-sass is a wrapper for libsass (C++ library), which I think is what Tom used.

This has relevant links https://github.com/sass/libsass

Reply:
perhaps we should add `libsass-python` as a dependency in frontend and use it from there (or even make it part of the Docker build), I think that would be the easiest to use.

Reply:
Agree, as part of the docker build is most convenient